_stbt/core.py

```diff
@@ -933,9 +933,6 @@ def _match_all(self, image, frame, match_parameters, region):
 
         region = Region.intersect(_image_region(frame), region)
 
-        # Remove this copy once we've confirmed that doing so won't cause
-        # push_sample to doodle on the frame later.
-        frame = frame.copy()
```

Comment:
Nice, earlier in the pull request I was wondering if you were going to get around to removing this copy. Should we add a test that we don't write on the frame? Something like:

```
get_frame
sleep
look at the frame
```

Reply:
Good idea.